src/pages/filter/ui/selected-group-items.tsx

```diff
@@ -0,0 +1,50 @@
+import { ReactElement } from 'react'
+import Checkbox from 'react-three-state-checkbox'
+import { observer } from 'mobx-react-lite'
+
+import { IVariantList } from '@declarations'
+import filterStore from '@store/filter'
+
+interface IProps {
+  chunks: any[]
+  currentPage: number
```

Comment:
You only need 'current page' for the index to access the item. Why not pass the actual item of the 'chunks'?

Reply:
Great point, I have fixed